components/tidb_query/src/codec/error.rs

```diff
@@ -207,4 +207,10 @@ impl From<codec::Error> for Error {
     }
 }
 
+impl From<tidb_query_datatype::DataTypeError> for Error {
+    fn from(err: tidb_query_datatype::DataTypeError) -> Self {
+        box_err!("invalid schema: {:?}", err.to_string())
```

Comment:
Why `schema`?

Reply:
Because schema in tikv is `[FieldType]`, where collation and FieldTypeTp store. So I think this kind of error is caused by the invalid schema.

Reply:
okey